### PR TITLE
AQL PRUNE: add note about edge for starting vertex

### PR DESCRIPTION
**Pull request by Dronplane**
Added small note for PRUNE condition evaluation at starting vertex. Inspired by customer ticket https://arangodb.atlassian.net/browse/ES-715

### CODE REVIEW

File: 3.8/aql/graphs-traversals.md [hunk 1]
```diff
@@ -220,6 +220,11 @@ to be checked during a search. So it allows to improve query performance and red
 the amount of overhead generated by the query. Pruning can be executed on the
 vertex, the edge and the path and any variable defined before.
 
+**Note**: While building a PRUNE condition for the 'edge' variable one should keep in mind that for starting
+vertex the edge is null. So having PRUNE condition like (edge.label != 'foo') could undesirable be
```
Comment (goedderz) [inline 3.8/aql/graphs-traversals.md:224]:
```suggestion
vertex the edge is null. So having `PRUNE` condition like `edge.label != 'foo'` could undesirably be 
```

File: 3.8/aql/graphs-traversals.md [hunk 2]
```diff
@@ -220,6 +220,11 @@ to be checked during a search. So it allows to improve query performance and red
 the amount of overhead generated by the query. Pruning can be executed on the
 vertex, the edge and the path and any variable defined before.
 
+**Note**: While building a PRUNE condition for the 'edge' variable one should keep in mind that for starting
+vertex the edge is null. So having PRUNE condition like (edge.label != 'foo') could undesirable be 
+true and terminate traversal too early. Construction like (!IS_NULL(edge) AND edge.label != 'foo') should be
```
Comment (goedderz) [inline 3.8/aql/graphs-traversals.md:225]:
```suggestion
true and terminate the traversal too early. A construction like `(!IS_NULL(edge) AND edge.label != 'foo')` should be
```

File: 3.8/aql/graphs-traversals.md [hunk 3]
```diff
@@ -220,6 +220,11 @@ to be checked during a search. So it allows to improve query performance and red
 the amount of overhead generated by the query. Pruning can be executed on the
 vertex, the edge and the path and any variable defined before.
 
+**Note**: While building a PRUNE condition for the 'edge' variable one should keep in mind that for starting
```
Comment (goedderz) [inline 3.8/aql/graphs-traversals.md:223]:
```suggestion
**Note**: While building a `PRUNE` condition for the *edge* variable one should keep in mind that for starting
```

**Review by goedderz — APPROVED** [review]

**Comment by Simran-B** [comment]
Rephrased and moved to the bottom of the PRUNE section:
https://deploy-preview-577--zealous-morse-14392b.netlify.app/docs/3.8/aql/graphs-traversals.html#pruning

**Pull request merged**